### PR TITLE
cc_ggplot2_cheatsheet

### PR DESCRIPTION
**Pull request by 021771618**

### CODE REVIEW

File: CC_ggplot2_cheatsheet.Rmd [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# ggplot2_cheatsheet
+
+Xiaorui Zhang
+# 1. Some Advanced Techs from ggplot2
```
Comment (shauryamalik) [inline CC_ggplot2_cheatsheet.Rmd:4]: As per 2.2.3 (https://jtr13.github.io/cc21/github-submission-instructions.html) do not use single # anywhere other than first line of Rmd file. Please remove the same from line 4.

**Review by shauryamalik — CHANGES_REQUESTED** [review]
As per 2.2.3 (https://jtr13.github.io/cc21/github-submission-instructions.html) do not use single # anywhere other than first line of Rmd file. Please remove the same from line 4 as well as lines 142-144.

File: CC_ggplot2_cheatsheet.Rmd (hunk 1 above)
Comment (021771618) [reply CC_ggplot2_cheatsheet.Rmd:4]: ok, I got it. Thank you very much!

File: CC_ggplot2_cheatsheet.Rmd [hunk 2]
```diff
@@ -0,0 +1,169 @@
+# ggplot2_cheatsheet
+
+Xiaorui Zhang
+## 1. Some Advanced Techs from ggplot2
```
Comment (kennylee15) [inline CC_ggplot2_cheatsheet.Rmd:4]: Please add one more line here between the name and double hash tags.

File: CC_ggplot2_cheatsheet.Rmd [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+## [, 6]	wt	Weight (1000 lbs)
+## [, 7]	qsec	1/4 mile time
+## [, 8]	vs	Engine (0 = V-shaped, 1 = straight)
+## [, 9]	am	Transmission (0 = automatic, 1 = manual)
+## [,10]	gear	Number of forward gears
+## [,11]	carb	Number of carburetors
+head(mtcars)
+```
+
+### 1.2 Facet
```
Comment (kennylee15) [inline CC_ggplot2_cheatsheet.Rmd:24]: Please stick to hashtags as opposed to using both hashtags and numbered chapters. When rendered, the hashtags will automatically be numbered as subsections.

**Review by kennylee15 — COMMENTED** [review]
Please make a few more stylistic changes. I left in-line comments. Thanks!

**Review by yaojiaye00 — CHANGES_REQUESTED** [review]
Hi, just a reminder that you did not fix all the problem mentioned before.
Please watch out for the following addtional points to make changes:
1.The file name can only contain lowercase letters, meaning you should use cc instead of CC
2.Every hashtag header should not be followed by numbers, just use ## Subheading.
3.Better to include the link to internal file in Rmd file (Optional). (e.g.  some explanations 
  and link:   resources/<project_name>/xxx.pdf)
Thanks!

**Comment by 021771618** [comment]
Hi,
I have fixed all problems mentioned before except optional one. Thank you.

**Review by yaojiaye00 — CHANGES_REQUESTED** [review]
Hi, maybe you still did not change the file name into lowercase letters.*(The Rmd file name, not the project name)
Thanks!

**Comment by 021771618** [comment]
Hi,
I changed the file name right now. Thank you!

File: cc_ggplot2_cheatsheet.Rmd [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+  ## Reduce the bar width to 0.1
+  geom_bar(width = 0.1) +
+  coord_polar(theta = "y") +
+  ## Add a continuous x scale from 0.5 to 1.5
+  scale_x_continuous(limits = c(0.5,1.5))
+```
+
+## Conclusion
+## This is only a little part of ggplot2, and then in my additional resources, compared with ggplot2, 
+## everyone can learn the visualization tools from Python if needed.
```
Comment (shauryamalik) [inline cc_ggplot2_cheatsheet.Rmd:145]: Are these double hashtags intentional? If not, double hashtags are only needed for Conclusion heading and can be removed from lines 144-145.

**Review by shauryamalik — CHANGES_REQUESTED** [review]

File: cc_ggplot2_cheatsheet.Rmd (hunk 4 above)
Comment (021771618) [reply cc_ggplot2_cheatsheet.Rmd]: I did it right now. Thank you!

**Comment by yaojiaye00** [comment]
Thanks for your contribution!

**Comment by 021771618** [comment]
Thank you for your advice!

**Pull request merged**